### DISCUSSION TITLE
Toggle Hide/Show All Annotations Hotkey

### DISCUSSION THREAD

**Discussion by NRHGDW** [sumatrapdfreader/sumatrapdf#3071] [Ideas]
Please add a toggle for hide/show all annotations, as hotkey if possible.
Effect:
Hides or shows all annotations for this session

Implementation examples:
Microsoft Edge: open PDF in MS Edge, click gear on top right, click hide all annotations
Acrobat DC: open PDF in acrobat dc, use the rather annoying hotkeys: ctrl+8 / ctrl+shift+8

Example purpose:
Sometimes need to make notes, annotations, and so on, but later need to see the original without all the scribbling.

If this already exists:
Please add to documentation at https://www.sumatrapdfreader.org/docs/Editing-annotations

**Comment by kjk** [comment]
Opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3266 to track this

**Comment by GitHubRulesOK** [comment]
See above function has now been added as 3 commands on off & toggle

>Per-tab toggle to hide/show PDF annotations. When hidden, page contents
and form widgets are still rendered but annotations are skipped.
